activesupport/test/encrypted_file_test.rb

```diff
@@ -13,7 +13,7 @@ class EncryptedFileTest < ActiveSupport::TestCase
     File.write(@key_path, ActiveSupport::EncryptedFile.generate_key)
 
     @encrypted_file = ActiveSupport::EncryptedFile.new \
-      content_path: @content_path, key_path: @key_path, env_key: "CONTENT_KEY"
+      content_path: @content_path, key_path: @key_path, env_key: "CONTENT_KEY", raise_if_missing_key: true
```

Comment:
Let's replace `\` with parens here and in the other test. This makes @rafaelfranca sleep better at night 😄